src/encrypt.ts

```diff
@@ -0,0 +1,57 @@
+import * as crypto from "crypto";
+import * as t from "io-ts";
+
+const IV_LENGTH = 16;
+const AES_LENGTH = 16;
+
+const EncryptTuple = t.interface({
+  // Hybrid encrypted result (Base64)
+  encryptedOutput: t.string,
+
+  // random AES string (Base64)
+  iv: t.string,
+
+  // AES Key encrypted with RSA public key (Base64)
+  rsaAesKey: t.string
+});
+
+export type EncryptTuple = t.TypeOf<typeof EncryptTuple>;
+
+export function hybridEncrypt(input: string, rsaPubKey: string): EncryptTuple {
```

Comment:
```suggestion
export function toEncryptedPayload(rsaPubKey: string, input: string): EncryptedPayload {
```

naming: try not to leak implementation details ("hybrid")

order of parameters: always try put the "most variable" input last since this favor partial application with curried methods (https://en.wikipedia.org/wiki/Tacit_programming) ie.

```ts
const encrypt = curry(toEncryptedPayload)(MY_RSA_KEY);
const encryptedPayload1 = encrypt("foo"); // <--- note we don't use rsa key here
const encryptedPayload2 = encrypt("bar"); // <--- note we don't use rsa key here
```

this would be not possible if you put `input` first